charts/immich/templates/server/deployment.yaml

```diff
@@ -3,40 +3,31 @@ apiVersion: apps/v1
 metadata:
   name: {{ template "common.fullname" . }}-server
   labels:
-    app.kubernetes.io/name: {{ template "common.name" . }}
-    app.kubernetes.io/instance: {{ .Release.Name }}
-    app.kubernetes.io/component: "immich-server"
+    app.kubernetes.io/component: "server"
     {{- include "common.labels" . | nindent 4 }}
 spec:
   revisionHistoryLimit: 0
   replicas: {{ .Values.server.replicas }}
-  strategy:
-    type: Recreate
   selector:
     matchLabels:
-      app.kubernetes.io/name: {{ template "common.name" . }}
-      app.kubernetes.io/instance: {{ .Release.Name }}
-      app.kubernetes.io/component: "immich-server"
+      app.kubernetes.io/component: "server"
+      {{- include "common.selectorLabels" . | nindent 6 }}
   template:
     metadata:
       labels:
-        app.kubernetes.io/name: {{ template "common.name" . }}
-        app.kubernetes.io/instance: {{ .Release.Name }}
-        app.kubernetes.io/component: "immich-server"
-        {{- include "common.labels" . | nindent 8 }}
+        app.kubernetes.io/component: "server"
+        {{- include "common.selectorLabels" . | nindent 8 }}
     spec:
       securityContext: {{ toYaml .Values.server.podSecurityContext | nindent 8 }}
       containers:
         - name: server
           image: "{{ .Values.server.image.registry }}/{{ .Values.server.image.repository }}:{{ .Values.server.image.tag }}"
           imagePullPolicy: {{ .Values.server.image.pullPolicy }}
           env:
+            - name: TZ
+              value: {{ .Values.timezone }}
             - name: IMMICH_TELEMETRY_INCLUDE
-              value: "all"
-            - name: IMMICH_METRICS_PORT
-              value: "8081"
```

Comment:
This env variable was replaced when the api and microservices images were merged and this was the default port number anyway